core/src/main/java/org/geysermc/geyser/item/mappings/versions/MappingsReader.java

```diff
@@ -90,4 +92,21 @@ protected CustomRenderOffsets.OffsetXYZ getOffsetXYZ(JsonNode node, String offse
                 tmpNode.get("z").floatValue()
         );
     }
+
+    protected Set<String> getTags(JsonNode node) {
+        if (node == null || !node.isArray()) {
+            return null;
+        }
+
+        Set<String> temp = new HashSet<>();
+        for (JsonNode tag : node) {
+            if (tag != null && tag.isTextual()) {
+                String tagContent = tag.asText();
+                if (!tagContent.isEmpty()) {
+                    temp.add(tagContent);
+                }
```

Comment:
`isBlank` probably wouldn't hurt. Same with the logic in CustomItemRegistryPopulator